### ISSUE TITLE
navigate search results via keyboard 

### ISSUE THREAD

**Issue by egladman** [CWDG/open-source-club-website#103]
the user should be able to select list items all from the keyboard. This will be accomplished by using a JS library called "Mousetrap".

**Comment by egladman** [comment]
fixed: 25bb4425a66365b4b7d03fd1c18b1ed49b675432
